### PR TITLE
add my HW01

### PR DESCRIPTION
**Pull request by NeverovDS**
My first homework.In the first task, I implemented a Converter from time to the angle between the hands of the clock.In the second and third tasks, I wrote down the explanations according to the tasks.

### CODE REVIEW

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+hour = gets.to_i
+min = gets.to_i
+# this method turns the time into an angle between the watch strips
+def your_angel(hour, min)
```
Comment (vladhilko) [inline HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: can you use a better name? `your_angel` doesn't describe the task very well

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  an = (min * 6 - (hour * 60 + min) * 0.5).abs
```
Comment (vladhilko) [inline HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: can you use the whole name for `minute`? `min` is confusing a little bit

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  if an > 180
```
Comment (vladhilko) [inline HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: the same for `an`

Comment (vladhilko) [reply HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: A variable name should be clear, readable and understandable

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    360 - an .abs
+  else
+    an .abs
```
Comment (vladhilko) [inline HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: Does it work?
```suggestion
    an.abs
```

**Comment by vladhilko** [comment]
Please, click `resolve conversation` and respond to comments if the code has been corrected

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+hours = gets.to_i
+minutes = gets.to_i
+# this method turns the time into an angle between the watch strips
+def calc_angel(hours, minutes)
+  angel = (minutes * 6 - (hours * 60 + minutes) * 0.5).abs
```
Comment (vladhilko) [inline HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: Why don't you want to use something like this?
```suggestion
  angel = (minutes * 6 - (hours * 30 + minutes * 0.5)).abs
```

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+hours = gets.to_i
+minutes = gets.to_i
+# this method turns the time into an angle between the watch strips
+def calc_angel(hours, minutes)
```
Comment (vladhilko) [inline HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]:
```suggestion
def calc_angle(hours, minutes)
```

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb (hunk 2 above)
Comment (NeverovDS) [reply HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: +

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb (hunk 3 above)
Comment (NeverovDS) [reply HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: +

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb (hunk 4 above)
Comment (NeverovDS) [reply HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: It work)

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb (hunk 5 above)
Comment (NeverovDS) [reply HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: There is a difference?

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb (hunk 6 above)
Comment (NeverovDS) [reply HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: +

File: HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+def calc_angle(hours, minutes)
+  angel = (minutes * 6 - (hours * 60 + minutes) * 0.5).abs
+  if angel > 180
+    360 - angel.abs
+  else
+    angel.abs
+  end
+end
+
+puts "#{calc_angel(hours, minutes)} degrees"
```
Comment (vladhilko) [inline HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: Change the name everywhere, please
```suggestion
puts "#{calc_angle(hours, minutes)} degrees"
```

Comment (NeverovDS) [reply HW01/Dzmitry_Neverov/dzmitry_neverov_hw_01_t_01.rb]: +

**Pull request merged**